plugins/feeds/public/virustotal_apiv3.py

```diff
@@ -0,0 +1,71 @@
+import logging
+import re
+import json
+from datetime import timedelta, datetime
+
+from core import Feed
+from core.config.config import yeti_config
+from core.observables import Hash, File
+
+# Variable
+VTAPI = yeti_config.get('vt', 'key')
+headers = {"x-apikey": VTAPI}
+limit = 10
+params = {'limit': limit}
+regex = "[A-Fa-f0-9]{64}"  # Find SHA256
+
+
+class VirusTotalPriv(Feed):
+    default_values = {
+        "frequency": timedelta(minutes=5),
+        "name": "VirusTotalHuntingV3",
+        "source": "https://www.virustotal.com/api/v3/intelligence/hunting_notifications",
+        "description": "Feed of hunting for VirusTotal API v3",
+    }
+
+    settings = {
+            'vt_url_hunting_v3': {
+            'name': 'VT Url Hunting v3',
+            'description': 'Hunting feed for VT API v3'
+        }
+    }
+
+    def update(self):
+        if VTAPI:
+            self.source = "https://www.virustotal.com/api/v3/intelligence/hunting_notifications"
+            for index, item in self.update_json(params=params, headers=headers, key="data"):
+                self.analyze(item)
+        else:
+            logging.error("Your VT API key is not set in the config file!")
+
+    def analyze(self, item):
+        tags = []
+
+        # Convert data to json
+        json_string = item.to_json()
```

Comment:
why transfom item in json ? you access data by the name of rows like a dict

Reply:
corrected and using item directly